src/app/share-preview/components/share-preview/share-preview.component.html

```diff
@@ -69,10 +69,12 @@
 	</div>
 </div>
 <div class="share-preview-content-breadcrumbs">
-	<pr-breadcrumbs></pr-breadcrumbs>
+	<pr-breadcrumbs
+		(onBreadcrumbClicked)="goToFolderFromBreadcrumb($event)"
+	></pr-breadcrumbs>
 </div>
 <div class="share-preview-content" [ngClass]="{ navigating: isNavigating }">
-	@if (true) {
+	@if (ephemeralFolder) {
```

Comment:
Since `ephemeralFolder` isn't a boolean can we be more explicit about what we're checking here?  (e.g. `(ephemeralFolder !== null)`)

This will eventually be required by our linter, but for now we may as well follow the practice.